CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ appear at the top.
 ## [Unreleased][]
 
   * Your contribution here!
+  * [#397](https://github.com/capistrano/sshkit/pull/397): Fix net-ssh default options with net-ssh < 4.0.0 - [@shirosaki](https://github.com/shirosaki)
```

Comment:
I would rephrase the changelog entry so that people know exactly what error is fixed. Something like this:

> Fix NoMethodError `assign_defaults` with net-ssh older than 4.0.0